chart/kubeapps/values.yaml

```diff
@@ -132,6 +132,9 @@ ingress:
 ##
 frontend:
   replicaCount: 2
+  ## Add additional annotations to the frontend deployment
+  ##
+  annotations: {}
```

Comment:
So my only qualm here is that we're using `frontend.annotations` and `apprepository.annotations` to mean something which is really just annotations for the deployment, not the frontend (service, for eg). In this sense, `podAnnotations` would be specific and not need to change in the future.

Reply:
Let's just change this to podAnnotation, and move the annotations to the pod level instead of the deployment level.
This way we conform with what is being done in other charts, and for me it's an equally good solution.

I'll do this change when I have the time, might be able to do it today.

Reply:
Perfect, thanks @kristinnardal2

Reply:
I've done this, I also noticed an error while doing this, adding multiple annotations would have failed because the indentation wasn't correct. I have added a `nindent` to fix this issue.